<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="account_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Аккаунт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="throw_dice_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Кинуть дайс</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_main_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>На главную</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_questionnaire_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Создать анкету</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scroll_area">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>411</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
  /* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 

QScrollArea {
color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
}

QVBoxLayout {
  background-color: transparent;
}

QWidget {
  background-color: transparent;
}
</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>397</width>
       <height>397</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url('data/textures/background.jpg');
background-repeat: no-repeat; 
background-position: center;</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>account_btn</zorder>
   <zorder>throw_dice_btn</zorder>
   <zorder>to_main_btn</zorder>
   <zorder>create_questionnaire_btn</zorder>
   <zorder>scroll_area</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
